.projenrc.ts

```diff
@@ -95,11 +95,10 @@ project.testTask.prependSpawn(installHelm);
 
 project.compileTask.spawn(schemas);
 
-// so that it works on windows as well
-// default projen uses $(npm pack) which fails
-project.packageTask.reset();
```

Comment:
Nice! Thanks.

Reply:
Ah yes, that's not needed anymore apparently.